app.js

```diff
@@ -9,10 +9,13 @@ const session = require( 'express-session' )
 const models = require( './models/index' )
 const passport = require( './auth/passport' )
 const protectRoute = require( './auth/protectRoute' )
+const verifyJwt = require('./auth/verify-jwt')
```

Comment:
This name is very implementation dependent - consider a more generic name that abstracts our API authentication process, like `verifyToken` (although we're not really verifying the token - it seems more like we're verifying that the token exists?).